<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminWidget</class>
 <widget class="QWidget" name="AdminWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1296</width>
    <height>814</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="2,1,30">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>9</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="adduserbtn">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>45</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>45</height>
          </size>
         </property>
         <property name="text">
          <string> 添加用户</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteuserbtn">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>45</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>45</height>
          </size>
         </property>
         <property name="text">
          <string>删除用户</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="modifyBtn">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>45</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>45</height>
          </size>
         </property>
         <property name="text">
          <string>修改信息</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="flushBtn">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>45</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>45</height>
          </size>
         </property>
         <property name="text">
          <string>刷新信息</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exitBtn">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>45</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>45</height>
          </size>
         </property>
         <property name="text">
          <string>退出登录</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="serachuser">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>45</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>45</height>
          </size>
         </property>
         <property name="text">
          <string>查找用户</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit">
         <property name="minimumSize">
          <size>
           <width>360</width>
           <height>45</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>350</width>
           <height>45</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_1">
        <attribute name="title">
         <string>用户管理</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QTableView" name="user_tableView"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>数据</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QTableWidget" name="tableWidget_2">
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <column>
            <property name="text">
             <string>ID</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>姓名</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>性别</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>年龄</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>职务</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>联系方式</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>账户密码</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>权限</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
